charts/victoria-metrics-alert/values.yaml

```diff
@@ -308,6 +308,8 @@ server:
     #         cpu: 1
     #         memory: 500Mi
     #       controlledResources: ["cpu", "memory"]
+  ## Additional initContainers to initialize the pod
```

Comment:
please use '# -- ' in front of comment messages to be parsed correctly by helm docs and run 'make gen-docs' from a repo root

Reply:
done.